src/verifiers/documentStoreIssued/tokenRegistryContractInterface.ts

```diff
@@ -0,0 +1,79 @@
+import { constants, errors } from "ethers";
```

Comment:
why does this file live in this folder?

Reply:
I wanted to make sure you read this carefully. Thanks

Reply:
done